epitator/spacy_annotator.py

```diff
@@ -77,6 +87,6 @@ def annotate(self, doc):
                 if date_as_number < 1900:
                     ne_span.label = 'QUANTITY'
 
-        tiers['spacy.tokens'] = AnnoTier(token_spans)
-        tiers['spacy.nes'] = AnnoTier(ne_spans)
+        tiers['spacy.tokens'] = AnnoTier(token_spans, presorted=True)
+        tiers['spacy.nes'] = AnnoTier(ne_spans, presorted=True)
```

Comment:
In the InfectionAnnotator PR, I added these lines:

```python
tiers['spacy.noun_chunks'] = AnnoTier([
    SentSpan(chunk, doc) for chunk in spacy_doc.noun_chunks])
```
This should of course be adjusted for this new way of doing stuff.

Reply:
Would it be tough for you to adapt this addition to this commit, so that a noun chunk tier is created? That way, I can just pull these changes into the InfectionAnnotator branch without having to edit the method myself.